collect/collect.go

```diff
@@ -270,7 +270,7 @@ func (i *InMemCollector) checkAlloc() {
 		WithField("num_traces_sent", len(tracesSent)).
 		WithField("datasize_sent", totalDataSizeSent).
 		WithField("new_trace_count", i.cache.GetCacheCapacity()).
-		Logf("evicting large traces early due to memory overage")
+		Logf("evicting large traces early due to memory overage. Evicted traces will be processed and if selected by sampling, sent to Honeycomb.")
```

Comment:
Same as this one. Something like `Making trace decision for evicted large traces early due to memory overage`

Reply:
```suggestion
		Logf("Making some trace decisions early due to memory overrun.")
```